include/lemlib/devices/trackingWheel.hpp

```diff
@@ -44,50 +45,50 @@ class TrackingWheel {
          * @brief Create a tracking wheel with a custom encoder
          *
          * @param encoder unique ptr to the custom encoder to be used
-         * @param diameter the diameter of the wheel, in inches
-         * @param offset distance between the wheel and the tracking center, in inches
+         * @param diameter the diameter of the wheel
+         * @param offset distance between the wheel and the tracking center
          */
-        TrackingWheel(std::shared_ptr<Encoder> encoder, float diameter, float offset);
+        TrackingWheel(std::shared_ptr<Encoder> encoder, Length diameter, Length offset);
         /**
          * @brief Create a new motor-encoder tracking wheel
          *
          * @param motors pointer to the motor group to be used for the tracking wheel
-         * @param diameter the diameter of the wheel, in inches
-         * @param offset distance between the wheel and the tracking center, in inches
-         * @param rpm of the rpm of the wheels the motor group is driving
+         * @param diameter the diameter of the wheel
+         * @param offset distance between the wheel and the tracking center
+         * @param speed the rpm of the wheels the motor group is driving
```

Comment:
```suggestion
         * @param speed the angular velocity of the wheels the motor group is driving (e.g 360_rpm)
```
Remove rpm from parameter description